### PR TITLE
Fix @track and @wire conflict, simplify component

### PR DESCRIPTION
**Pull request by kevinv11n**

### CODE REVIEW

File: force-app/main/default/lwc/wireGetObjectInfo/wireGetObjectInfo.js [hunk 1]
```diff
@@ -1,20 +1,14 @@
-import { LightningElement, wire, track } from 'lwc';
+import { LightningElement, wire } from 'lwc';
 import { getObjectInfo } from 'lightning/uiObjectInfoApi';
 
 export default class WireGetObjectInfo extends LightningElement {
-    objectApiNameInputValue = 'Account';
     objectApiName;
-    @track objectInfo;
```
Comment (kevinv11n) [inline force-app/main/default/lwc/wireGetObjectInfo/wireGetObjectInfo.js]: Below we `@wire` onto `objectInfo` so we shouldn't have this.

File: force-app/main/default/lwc/wireGetObjectInfo/wireGetObjectInfo.js [hunk 2]
```diff
@@ -1,20 +1,14 @@
...
 
     @wire(getObjectInfo, { objectApiName: '$objectApiName' })
     objectInfo;
 
-    objectNameChange(event) {
```
Comment (kevinv11n) [inline force-app/main/default/lwc/wireGetObjectInfo/wireGetObjectInfo.js]: No need to collect the value as it's entered. We can get it in the button click handler.

**Comment by muenzpraeger** [comment]
Please run prettier, @kevinv11n

**Comment by kevinv11n** [comment]
It's odd the `pre-commit` hook didn't catch this.

**Comment by kevinv11n** [comment]
I rebased and squashed onto `trailheadapps:prerelease/summer19` per your PR target change.

**Pull request merged**